R/utils.R

```diff
@@ -227,10 +228,16 @@ save_chunkdata <- function(chunkdata, write_inputs = FALSE, outputs_dir =
   # Create directory if necessary, and remove any previous outputs
   dir.create(outputs_dir, showWarnings = FALSE, recursive = TRUE)
   unlink(file.path(outputs_dir, "*.csv"))
+  dir.create(xml_dir, showWarnings = FALSE, recursive = TRUE)
+  unlink(file.path(xml_dir, "*.xml"))
 
   for(cn in names(chunkdata)) {
     cd <- chunkdata[[cn]]
-    if(!isTRUE(identical(NA, cd))) {   # NA means an optional file that wasn't found
+    if(FLAG_XML %in% get_flags(cd)) {
+      # TODO: worry about absolute paths?
```

Comment:
What do you mean by "worry" here?  Absolute paths are easy to implement, so I think we should make a decision.  Either the absolute path issue is important enough to deal with now, or we drop the TODO and just deal with it if and when it ever comes up.